toolkit/docs/formats/imageconfig.md

```diff
@@ -251,6 +251,10 @@ A sample KernelCommandLine enabling a basic IMA mode and passing two additional
 },
 ```
 
+### HidepidDisable
+
+An optional flag that removes the hidepid option from /proc. Hidepid prevents proc ids from being visible to all users.
```

Comment:
Since this is a user-facing piece of documentation, I'd suggest a few changes:
- Surround "hidepid" in quotes, or use _italic text_, or put it in a `code block`.
- Let's use "IDs" - [with a capital "ID" and a small "s"](https://english.stackexchange.com/questions/212040/is-the-plural-form-of-id-spelled-ids-or-id).
- Maybe also put `/path` in a code block.

Not sure about the last one. Where is our documentation writer?

Reply:
updated!